flexmeasures/data/schemas/sensors.py

```diff
@@ -242,11 +242,22 @@ def _deserialize(
 
         elif isinstance(value, str):
             try:
+                try:
+                    float(value)
+                    return value
+                except ValueError:
+                    pass
                 return ur.Quantity(value).to(self.to_unit)
             except DimensionalityError as e:
                 raise FMValidationError(
                     f"Cannot convert value `{value}` to '{self.to_unit}'"
                 ) from e
+        elif (
+            isinstance(value, Sensor)
+            or isinstance(value, int)
+            or isinstance(value, float)
+        ):
+            return value
```

Comment:
I don't think we'll need changes to this schema for now. SoC values passed to the new field should be sensor descriptions (`{"sensor": <id>}`) or strings containing a unit that is convertible to MWh (e.g. `"50 kWh"`).

Reply:
The flex model gets SoC sensor and not a dictionay . So I still need to have a check for sensor as value.

Reply:
What do you think about this change?

Reply:
> The flex model gets SoC sensor and not a dictionay . So I still need to have a check for sensor as value.

I'm not able to verify this. When I replace `return value` with `raise ValueError` all tests still pass. To me it makes sense that the `_deserialize` method is only getting called on a flex-model with serialized fields.

Reply:
I expanded a test that made it clear for me that the new deserialization case is indeed not needed.